runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/hotcold/HotColdTemperature.java

```diff
@@ -93,11 +94,12 @@ public static HotColdTemperature getFromTemperatureSet(final Set<HotColdTemperat
 			return null;
 		}
 
+		final String messageStart = Text.fromCSV(message).get(0);
```

Comment:
Shouldnt this also check if the split is not empty? just in case

Reply:
This method already does an early return if the message doesn't start with `DEVICE_USED_START_TEXT` ("The device is "), so there will always be something in the first split.

Reply:
Alright